http-server.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const httpServer = require('http-server');
+
+const root = path.join(__dirname, '.');
+const port = process.env['PORT'] || 8081;
+const host = '127.0.0.1';
```

Comment:
Can we setup httpServer for remote access?
It seems https should be applied due to 'Cross-Origin-Opener-Policy' and 'Cross-Origin-Embedder-Policy', but https requires SSL certificate.

Reply:
You can run server script on the remote machine and open ssh-tunnel to keep this connection like local.
Or do you prefer run this script on domain?